tests/test_nodeset_target.py

```diff
@@ -0,0 +1,18 @@
+import numpy
+
+from neurodamus.core.nodeset import NodeSet
+from neurodamus.target_manager import NodesetTarget
```

Comment:
`python3-base` tests still don't run forked and we have had hard-to-debug issues around that. I would suggest we do these imports inside the test test function

Reply:
Sure, I've moved them.